libres/lib/enkf/hook_manager.cpp

```diff
@@ -228,3 +222,51 @@ hook_manager_iget_hook_workflow(const hook_manager_type *hook_manager,
 int hook_manager_get_size(const hook_manager_type *hook_manager) {
     return vector_get_size(hook_manager->hook_workflow_list);
 }
+
+bool Runpath::operator<(const Runpath &other) const {
+    if (iter < other.iter)
+        return true;
+    if (iens < other.iens)
+        return true;
+    return false;
```

Comment:
```suggestion
    return iter < other.iter || iens < other.iens;
```

Reply:
Oups, that is indeed a correct transformation of the logic there, but the intention was lexiographical. Fixed it.